.github/workflows/gitlab-push.yml

```diff
@@ -16,6 +16,37 @@ env:
   PR_NUMBER: ${{ github.event.pull_request.number }}
 
 jobs:
+  push_to_gitlab_improved:
```

Comment:
Does this create an MR in GitLab whenever a GitHub PR is merged? is it guaranteed that the commit cherry picked is the one from the PR just merged? I can imagine some timing issues where another PR gets merged by the time the cherry picking is done